orca-core/src/main/java/com/netflix/spinnaker/orca/notifications/scheduling/OldPipelineCleanupPollingNotificationAgent.java

```diff
@@ -86,61 +79,50 @@ public Boolean call(Execution execution) {
     return 0;
   };
 
-  private Clock clock = Clock.systemDefaultZone();
+  Clock clock = Clock.systemDefaultZone();
```

Comment:
why no private / private final?

Reply:
Whoop. Meant to keep the private. Can't be final because of tests.

Reply:
Can't it got in the autowired constructor? We have a `Clock` bean in the spring context.